### ISSUE TITLE
Type error on Craft 3

### ISSUE THREAD

**Issue by khalwat** [webmenedzser/craft-reporter#24]
This line: <https://github.com/webmenedzser/craft-reporter/blob/master/src/Reporter.php#L60> 

```php
    public string $schemaVersion = '1.2.0';
```

needs to be:

```php
    public $schemaVersion = '1.2.0';
```

to work with Craft 3. The same with the types of the other properties.

You're going to need a separate branch of the plugin for Craft 3 / Craft 4

**Comment by Saboteur777** [comment]
Thanks for the heads up, I will push out an update tomorrow. 

p.s.: Are you going to finally try out Craft Report? :D

**Comment by khalwat** [comment]
I was helping someone with a problem in Discord :)

**Comment by Saboteur777** [comment]
Damn... 😁 

This will be my first thing to do tomorrow.

**Comment by Saboteur777** [comment]
2.0.0, 2.0.1, 2.0.2 and 2.0.3 were pulled from Packagist to prevent this from happening. 
Also released 2.1.0 which should fix this issue (by requiring Craft 4+).

@khalwat What do you think?

**Comment by Saboteur777** [comment]
As @khalwat gave a :+1: on it, I consider this as solved.
